src/utils/pathConflict.ts

```diff
@@ -29,7 +29,8 @@ export default async function pathConflict(targetDir: string, binaryName: string
 
     return [];
   }
-  const proposedVersion = await getVersion(referencePath, binaryName);
+  // We don't ship nerdctl, just an unversioned stub; so hard-wire a truthy value.
+  const proposedVersion = binaryName === 'nerdctl' ? '1.2.3' : await getVersion(referencePath, binaryName);
```

Comment:
Should this do `docker` too?

Reply:
Yes, I see this error message in the console:
`Trying to determine version, can't get output from /Users/ericp/workspace/rancher/desktop/resources/darwin/bin/docker`

Reply:
Going to create another issue.